include/tables/INodeTable.h

```diff
@@ -50,9 +50,9 @@ struct INodeRow {
     return mNumUserXAttrs > 0 ||  mNumSysXAttrs > 0;
   }
 
-  bool is_equal(ProjectRow proj){
-    return proj.mInodeName == mName && proj.mInodeParentId == mParentId 
-            && proj.mInodePartitionId == mPartitionId;
+  //assumption is project dirs have parentId = partitionId
```

Comment:
maybe no need for the is_equal here at all and just put the check explicitly in the Reindexer

Reply:
done